functions/src/healthSummary/healthSummaryData.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We might want to look into why this is triggered as we actually use all of them in the project. Maybe some digging like https://github.com/typescript-eslint/typescript-eslint/issues/2961 might help?

Reply:
I just didn't care enough yet - I will create an issue for that.